content/systems/filecoin_vm/message/_index.md

```diff
@@ -10,41 +10,139 @@ dashboardTests: 0
 # VM Message - Actor Method Invocation
 
 A message is the unit of communication between two actors, and thus the primitive cause of changes
-in state. A message combines:
+in state. When referring to a Message in the context of Filecoin and unless otherwise defined, we refer to a _Transaction Message_. A message combines:
```

Comment:
What other kind of message is there?

Reply:
Hmm. The message in blocks is very nearly the same concept as message passed between actors - close enough that I think it's ok to use the same term. And an exit code is associated with this same message concept - the internal and top-level ones both have exit codes.

My concern is that this sentence introduces a new term, "transaction", that isn't defined or used anywhere else in the spec. (The Runtime state accessor uses the term, but it means something different there).